modules/labs-react/header/stories/stories.tsx

```diff
@@ -481,20 +482,44 @@ storiesOf('Labs/Header/React', module)
     </div>
   ))
   .add('Search Form', () => (
```

Comment:
Are there visual tests for this component? I couldn't find any so I wasn't sure how to update them.

Reply:
Good question. These are just the stories that get rendered in the docs. There are no visual regressions for the Header components.